rre/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM maven:3.6.3-adoptopenjdk-11
+FROM maven:3-eclipse-temurin-11
```

Comment:
Can you provide context on why this one versust he other one?  Just curious!

Reply:
`adoptopenjdk` became `eclipse-temurin` - https://blog.adoptopenjdk.net/2021/08/goodbye-adoptopenjdk-hello-adoptium/

The other subtle change here is moving from specific Maven version 3.6.3 - to the latest 3.x maven version. 3.8.4 is the current Maven 3.x version. This gives up some reproducibility for making sure latest Maven 3.x docker image is used. There is a `3.8.4-eclipse-temurin-11` tag if would prefer locking the Maven 3.x version.

Reply:
awesome!